subprojects/diagnostics/src/integTest/groovy/org/gradle/api/plugins/ProjectReportsPluginIntegrationSpec.groovy

```diff
@@ -28,22 +29,39 @@ class ProjectReportsPluginIntegrationSpec extends AbstractIntegrationSpec {
     def "produces report files"() {
         when:
         succeeds("projectReport")
+
         then:
         file("build/reports/project/dependencies.txt").assertExists()
         file("build/reports/project/properties.txt").assertExists()
         file("build/reports/project/tasks.txt").assertExists()
+        file("build/reports/project/dependencies").assertIsDir()
     }
 
     def "produces report files in custom directory"() {
         given:
         buildFile << """
             projectReportDirName = "custom"
         """
+
         when:
         succeeds("projectReport")
+
         then:
         file("build/reports/custom/dependencies.txt").assertExists()
         file("build/reports/custom/properties.txt").assertExists()
         file("build/reports/custom/tasks.txt").assertExists()
+        file("build/reports/custom/dependencies").assertIsDir()
+    }
+
+    @Unroll
+    def "prints link to default #task"(String task) {
+        when:
+        succeeds(task)
+
+        then:
+        outputContains("See the report at:")
+
+        where:
+        task << ["taskReport", "propertyReport", "dependencyReport", "htmlDependencyReport"]
     }
```

Comment:
Could you also provide a test that explicitly sets outputFile = null for these tasks and asserts that the output does not contain the "See the report at:" message?